zio-http/src/test/scala/zio/http/codec/RichTextCodecSpec.scala

```diff
@@ -250,6 +250,10 @@ object RichTextCodecSpec extends ZIOHttpSpec {
         assertTrue(success(123) == codec.decode("123--")) &&
         assertTrue(codec.decode("4123").isLeft)
       },
+      test("With error message") {
+        val codec = RichTextCodec.literal("123").withError("Not 123")
+        assertTrue(codec.decode("678") == Left("(Expected, but did not find: Paragraph(Code(“1”,Inline)), Not 123)"))
```

Comment:
This is rendering poorly, perhaps because `describe` returns `Doc`, that should first be properly converted into a string.

Reply:
The problem is actually the `|` operator, that chains error messages this way explicitly.